modules/ossm-rn-fixed-issues.adoc

```diff
@@ -41,18 +41,22 @@ The following issues been resolved in the current release:
 
 * link:https://issues.redhat.com/browse/MAISTRA-2401[MAISTRA-2401] CVE-2021-3586 servicemesh-operator: NetworkPolicy resources incorrectly specified ports for ingress resources. The NetworkPolicy resources installed for {Productname} did not properly specify which ports could be accessed. This allowed access to all ports on these resources from any pod. Network policies applied to the following resources are affected:
 
```

Comment:
```suggestion
+
```
Adding a plus to this line indents the following list relative to the preceding bulleted release note.

Reply:
Then, lines 44 through 50 need only one `*` per item.

Reply:
https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#ex-complex